openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -21,6 +21,7 @@
 
 from ccx_keys.locator import CCXLocator
 
+THUMBNAIL_DIMENSIONS = (375, 200)
```

Comment:
To clarify, on default OpenedX installations course images can be a _maximum_ of the following sizes:

home page: 277 x 148
courses page: 277 x 148
course about pages: 338 x 180
dashboard: 197 x 105

Where is the 375 x 200 coming from, @ormsbee?